docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/BinaryClassification.ttinclude

```diff
@@ -113,6 +113,8 @@ namespace Samples.Dynamic.Trainers.BinaryClassification
             Console.WriteLine($"Negative Recall: {metrics.NegativeRecall:F2}");
             Console.WriteLine($"Positive Precision: {metrics.PositivePrecision:F2}");
             Console.WriteLine($"Positive Recall: {metrics.PositiveRecall:F2}");
+			Console.WriteLine();
+			Console.WriteLine(metrics.ConfusionMatrix.GetFormattedConfusionTable());
```

Comment:
You used tabs instead of spaces. Make sure you are using spaces, otherwise it could look differently.

Reply:
Also I think it would be cleaner if you add a new line character to the previous line:
```charp
Console.WriteLine($"Positive Recall: {metrics.PositiveRecall:F2}\n");
Console.WriteLine(metrics.ConfusionMatrix.GetFormattedConfusionTable());
```